source/routing/specifying-a-routes-model.md

```diff
@@ -1,120 +1,39 @@
-would define an `route:photos` object:
-
-```app/routes/photos.js
-export default Ember.Route.extend({
-  model() {
-    return [{
-      title: 'Tomster',
-      url: 'http://emberjs.com/images/about/ember-productivity-sm.png'
-    }, {
-      title: 'Eiffel Tower',
-      url: 'http://emberjs.com/images/about/ember-structure-sm.png'
-    }];
-  }
+```app/router.js
+Router.map(function() {
+  this.route('favoritePosts');
 });
 ```
 
-### Asynchronously Loading Models
-
-In the above example, the model data was returned synchronously from the
-`model` hook. This means that the data was available immediately and
-your application did not need to wait for it to load, in this case
-because we immediately returned an array of hardcoded data.
-
-Of course, this is not always realistic. Usually, the data will not be
-available synchronously, but instead must be loaded asynchronously over
-the network. For example, we may want to retrieve the list of photos
-from a JSON API available on our server.
-
-In cases where data is available asynchronously, you can just return a
-promise from the `model` hook, and Ember will wait until that promise is
-resolved before rendering the template.
-
-If you're unfamiliar with promises, the basic idea is that they are
-objects that represent eventual values. For example, if you use jQuery's
-`getJSON()` method, it will return a promise for the JSON that is
-eventually returned over the network. Ember uses this promise object to
-know when it has enough data to continue rendering.
-
-For more about promises, see [A Word on
-Promises](../../routing/asynchronous-routing/#toc_a-word-on-promises)
-in the Asynchronous Routing guide.
+To load a model for the `favoritePosts` route, you would use the `model` hook in
+the `posts` route handler:
 
-Let's look at an example in action. Here's a route that loads the most
-recent pull requests sent to Ember.js on GitHub:
-
-```app/routes/pull-requests.js
+```app/routes/favorite-posts.js
 export default Ember.Route.extend({
   model() {
-    return Ember.$.getJSON('https://api.github.com/repos/emberjs/ember.js/pulls');
+    return this.store.query('post', { favorite: true });
   }
 });
 ```
 
-While this example looks like it's synchronous, making it easy to read
-and reason about, it's actually completely asynchronous. That's because
-jQuery's `getJSON()` method returns a promise. Ember will detect the
-fact that you've returned a promise from the `model` hook, and wait
-until that promise resolves to render the `pullRequests` template.
-
-(For more information on jQuery's XHR functionality, see
-[jQuery.ajax](http://api.jquery.com/jQuery.ajax/) in the jQuery
-documentation.)
-
-Because Ember supports promises, it can work with any persistence
-library that uses them as part of its public API. You can also use many
-of the conveniences built in to promises to make your code even nicer.
+Typically, the `model` hook should return an Ember Data record.
```

Comment:
How about:

> Typically, the `model` hook should return an [Ember Data](../../models/) record, but it can also return any [promise](https://www.promisejs.org/) object, or a plain JavaScript object or array.